tgui/packages/tgui/interfaces/PaperSheet/Stamps.tsx

```diff
@@ -23,8 +23,11 @@ export const PaperSheetStamper: React.FC<PaperSheetStamperProps> = ({
   const [rotate, setRotate] = useState(0);
 
   const findStampPosition = (e: MouseEvent) => {
-    let rotating: boolean;
+    let rotating = false;
     const windowRef = document.querySelector('.Layout__content');
+    if (!windowRef) {
+      return;
+    }
```

Comment:
Stick this before `let rotating = false;`, as that's not required for the early return.